utils/zerotrie/src/options.rs

```diff
@@ -137,13 +149,27 @@ impl<S: ?Sized> crate::ZeroAsciiIgnoreCaseTrie<S> {
     pub(crate) const FLAGS: u8 = Self::OPTIONS.to_u8_flags();
 }
 
+/// Internal struct to power `get_strict`
+pub(crate) struct ZeroAsciiIgnoreCaseStrictTrie;
+
+impl ZeroTrieWithOptions for ZeroAsciiIgnoreCaseStrictTrie {
+    const OPTIONS: ZeroTrieBuilderOptions = ZeroTrieBuilderOptions {
+        phf_mode: PhfMode::BinaryOnly,
+        ascii_mode: AsciiMode::AsciiOnly,
+        capacity_mode: CapacityMode::Normal,
+        case_sensitivity: CaseSensitivity::IgnoreCase,
+        lookup_strictness: LookupStrictness::Strict,
```

Comment:
nit: can you do something like

```suggestion
        lookup_strictness: LookupStrictness::Strict,
        ..ZeroAsciiIgnoreCaseTrie::OPTIONS
```